### DISCUSSION TITLE
ionic  framework can use this library

### DISCUSSION THREAD

**Discussion by SadhanaKiwi** [alexandercerutti/passkit-generator#152] [Q&A]
can we use it's in ionic framework this library to create a pass for apple-wallet because this library many used two language typescript and JavaScript so  ionic also supported  typescript currently I am trying in ionic framework but it's not working if it's possible then we can try to create a generic pass and add them in apple-wallet   
if possible please suggest some method
thanks

**Comment by alexandercerutti** [comment]
Hey there @SadhanaKiwi ,
I've never used Ionic, so I don't know if this is possible.

Therefore I officially do not support it, but I'm available to discuss to see if there's something that could be done to support it.

This library is made to work with Node, so in the worst case you could setup a cloud function (firebase), a lambda (aws) or something else with passkit-generator on it and ping it to receive and serve an Apple Wallet pass.

Let me know!

**Reply by SadhanaKiwi** [reply]
@alexandercerutti  there is good idea we can use firebase to store a pass and download through them and we use node as a backend because we have another library in ionic this is work simply if you have pass.pkpass then it will easily add them in apple wallet   thanks I will try once I have find any good solution I will ping you thank you once again

**Reply by alexandercerutti** [reply]
You don't need to store it. You can just generate and serve it. We were discussing it in #150. Also, right yesterday I've edited serverless examples to show how you can make it download directly without passing through S3.

**Reply by SadhanaKiwi** [reply]
oh I got it  you already  mention there we will follow this approach  which you have to mention in Q&A thanks  for help
